test/server.test.js

```diff
@@ -32,6 +33,9 @@ injectGlobal`
     display: flex;
     justify-content: center;
   }
+  .should-be-included {
+    content: 'css-foo matches regex and this rule is swallowed (current bug)'
+  }
 `
```

Comment:
the css call returns a class name. Any nested selectors are prefixed with that class name for specificity.

Lets just pretend that that `css`...`` call returns `css-12345`. 

`.should-be-include { ... }` becomes `.css-12345 .should-be-included { ... }`